picard/ui/options/dialog.py

```diff
@@ -225,11 +226,34 @@ def show_attached_profiles_dialog(self):
         profile_dialog.activateWindow()
 
     def switch_profile(self, index):
+        if theme.is_dark_theme:
+            highlight = "#%s { color: lightyellow; background-color: #2F4F4F; }"
+        else:
+            highlight = "#%s { color: black; background-color: lightyellow; }"
         profile_id = self.ui.save_to_profile.currentData()
         config = get_config()
         config.setting.set_profile(profile_id)
+        settings = config.profiles[SettingConfigSection.SETTINGS_KEY]
+        if profile_id in settings:
+            profile_settings = settings[profile_id]
+        else:
+            profile_settings = {}
+
         for page in self.pages:
             page.load()
+            page_name = page.PARENT if page.PARENT in UserProfileGroups.SETTINGS_GROUPS else page.NAME
+            if page_name in UserProfileGroups.SETTINGS_GROUPS:
+                for opt in UserProfileGroups.SETTINGS_GROUPS[page_name]['settings']:
+                    for opt_field in opt.fields:
+                        try:
+                            obj = getattr(page.ui, opt_field)
+                        except AttributeError:
+                            continue
+                        if opt.name in profile_settings:
+                            style = highlight % opt_field
```

Comment:
The highlight is great, though it isn't obvious it is related to Options profiles.
Would it be possible to add a tooltip?